ACHNBrowserUI/ACHNBrowserUI/views/settings/SettingsView.swift

```diff
@@ -151,6 +159,46 @@ struct SettingsView: View {
         }
     }
     
+    private var notificationSection: some View {
+        Section(header: SectionHeaderView(text: "Notifications", icon: "clock")) {
+            HStack {
+                Toggle(isOn: $appUserDefaults.isShopOpenClose ) {
+                    Text("When shops open/close")
+                }
+                .onTapGesture {
+                    if !self.appUserDefaults.isShopOpenClose {
+                        self.getNotify(title: "ACHNBrowserUI", subtitle: "Nook’s Cranny store is open", hour: 8, minute: 0, isRepeated: true)
```

Comment:
Title of the notification should not be the repository name. If you want to use the name of the app, use „AC Helper“